spec/features/show_page/show_page_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+feature 'user adds a new animal' do
+  scenario 'user does not fill form and tries to submit' do
+    visit new_animal_path
+
+    click_button 'Add Animal'
+    expect(page).to have_content("Name can't be blank")
+    expect(page).to have_content("Body can't be blank")
+  end
+end
```

Comment:
Im a bit confused with this spec
- the name of the file says show, but this isnt for a show page
- you only are testing the sad path here, not the happy path as well 

Try to make this test more robust, and update the filename